src/services/twitter.js

```diff
@@ -405,7 +414,15 @@ class Twitter {
         const saved = await this.storage
           .twitterStatuses()
           .save(tweet);
-        this.publish(saved);
+
+        // Moment throws: value provided is not in a recognized RFC2822 or ISO format, that's why Date constructor is used.
+        const tweetDate = moment(new Date(tweet.date));
```

Comment:
can you show an example of tweet.date?

Reply:
@BrRenat I took an example value from here:
https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/tweet
 `"created_at": "Wed Oct 10 20:19:24 +0000 2018"`

Reply:
Changed, however I got some warnings for the specified date:
```
Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
```

Reply:
JFYI
If you know the format of an input string, you can use that to parse a moment.
moment('24/12/2019 09:15:00', "DD MM YYYY hh:mm:ss");